client/src/templates/Challenges/redux/completion-epic.js

```diff
@@ -34,16 +36,19 @@ import {
 function postChallenge(update, username) {
   const saveChallenge = postUpdate$(update).pipe(
     retry(3),
-    switchMap(({ points }) => {
+    switchMap(({ points, savedChallenges = undefined }) => {
```

Comment:
```suggestion
    switchMap(({ points, savedChallenges }) => {
```
`undefined` is the default.

Reply:
committed locally, will be resolved on next push